# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/videoconduit_unittests.cpp
# Commit: 1c3926cd7004
# Full Hash: 1c3926cd7004e7cba4b2265caee686d6201d21d2
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Move VideoConduit to call worker thread for mCall and related access. r=ng,bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112851
# ==============================================================================

diff -r 844625c1d96d -r 1c3926cd7004 media/webrtc/signaling/gtest/videoconduit_unittests.cpp
--- a/media/webrtc/signaling/gtest/videoconduit_unittests.cpp	Sun Apr 18 22:14:28 2021 +0200
+++ b/media/webrtc/signaling/gtest/videoconduit_unittests.cpp	Wed Apr 07 11:36:28 2021 +0200
@@ -51,6 +51,7 @@
             "")) {
     NSS_NoDB_Init(nullptr);
 
+    mVideoConduit->InitCall();
     mVideoConduit->SetLocalSSRCs({42}, {43});
   }
 
